AndelaOSP/Andela-Socials
# #166287132 Fix profile dropdown not showing

#### What Does This PR Do?
    This PR fixes the profile dropdown that is not showing

#### Description Of Task To Be Completed
    - add onclickoutside event listeners via package
    - change profile overflow to show

#### Any Background Context You Want To Provide?
   N/A
#### How can this be manually tested?
  Checkout to bg-fix-profile-dropdown-not-showing-166287132, click on the profile dropdown, you should see the profile

#### What are the relevant pivotal tracker stories?
[#166287132](https://www.pivotaltracker.com/story/show/166287132)

#### Screenshot(s)
N/A
